src/hotspot/cpu/riscv/gc/z/zGlobals_riscv.hpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2015, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think we'd better modify the copyright year of Oracle's.

Reply:
Done.